apps/dh/app-dh/project.json

```diff
@@ -35,6 +35,11 @@
             "input": "libs/ui-watt/src/assets",
             "glob": "**/*",
             "output": "assets/ui-watt"
+          },
+          {
+            "input": "libs/dh/shared/data-access-msw",
+            "glob": "mockServiceWorker.js",
```

Comment:
The reason why it's not inside of lib, is because it's auto-generated and should not be edited. But if you think it's better inside the lib folder, I just move it :)

Reply:
Isn't it possible to auto-generate it inside `/src/lib`?